### PR TITLE
More Data Dictionaries

### PR DESCRIPTION
**Pull request by pranavrajaram**
This will add data dictionaries for load_trades, load_injuries, load_depth_charts, and load_combine. Please let me know if I messed anything up, and happy new year!

### CODE REVIEW

**Comment by tanho63** [comment]
Looks awesome @pranavrajaram! Can you pull down the merge commit I just made (can probably just run `git pull` from your current), bump the version number in the DESCRIPTION, and add some NEWS updates as I did in NEWS.md?

File: data-raw/dictionary_combine.csv [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+draft_round,numeric,Round that player was drafted in
+draft_ovr,numeric,Pick number of player
+pfr_id,numeric,Pro-Football-Reference ID for player
+cfb_id,numeric,Sports Reference (CFB) ID for player
+player_name,character,Full name of player
+pos,character,Position of player
+school,character,College of player
+ht,numeric,Height of player (feet and inches)
+wt,numeric,Weight of player (lbs)
+forty,numeric,Player’s 40 yard dash time at combine (seconds)
```
Comment (tanho63) [inline data-raw/dictionary_combine.csv:14]: Ah! and it seems like you've used the wrong "apostrophe" here https://github.com/nflverse/nflreadr/runs/4680345266?check_suite_focus=true#step:8:108 - please make sure it's the straight apostrophe and not the angled one

File: data-raw/dictionary_depth_charts.csv [hunk 2]
```diff
@@ -0,0 +1,14 @@
+field,data_type,description
+season,numeric,4 digit number indicating to which season(s) the specified timeframe belongs to.
+week,numeric,Week in season that depth chart is for
+team,character,Team that depth chart belongs to
+season_type,character,REG' or 'POST' indicating if the depth chart belongs to regular or post season.
```
Comment (tanho63) [inline data-raw/dictionary_depth_charts.csv:5]: missing a quotation mark around REG

File: data-raw/dictionary_injuries.csv [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+team,character,Team of injured player
+week,numeric,Week that injury occurred
+gsis_id,numeric,Game Stats and Info Service ID: the primary ID for play-by-play data.
+position,character,Position of injured player
+full_name,character,Full name of player
+first_name,character,First name of player
+last_name,character,Last name of player
+report_primary_injury,character,Primary injury listed on official injury report
+report_secondary_injury,character,Secondary injury listed on official injury report
+report_status,character,Player’s status for game on official injury report
```
Comment (tanho63) [inline data-raw/dictionary_injuries.csv:13]: Non-ascii apostrophe

File: data-raw/dictionary_injuries.csv [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+practice_primary_injury,character,Primary injury listed on practice injury report
+practice_secondary_injury,character,Secondary injury listed on practice injury report
+practice_status,character,Player’s participation in practice
```
Comment (tanho63) [inline data-raw/dictionary_injuries.csv:16]: and again

**Review by tanho63 — CHANGES_REQUESTED** [review]
See notes about non-ascii apostrophes - that should fix cran-checks

**Comment by pranavrajaram** [comment]
thanks for all your help! I fixed the apostrophes and all the checks look to have passed, but I'm a bit confused about updating the description/news—Can I just change edit the files to include the new stuff in my fork of the repo and then pull request them back here? or is there something else that I'm missing

**Comment by pranavrajaram** [comment]
thanks for all your help! I fixed the apostrophes and the checks seem to have passed, but I'm not sure if I did the description & news update correctly.

**Review by tanho63 — APPROVED** [review]
LGTM!

**Comment by tanho63** [comment]
News update and DESCRIPTION updates look good to me! As a suggestion - you should probably check out a new branch instead of making the changes to "pranavrajaram/main" - see these helper functions in usethis that describe a good workflow https://usethis.r-lib.org/articles/pr-functions.html

**Comment by pranavrajaram** [comment]
thanks for the suggestion and all your help!

**Pull request merged**